examples/benchmarks/bubblebench/simple-tree/src/appState.ts

```diff
@@ -0,0 +1,76 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import {
+	type IAppState,
+	type IBubble,
+	type IClient,
+	makeBubble,
+	randomColor,
+} from "@fluid-example/bubblebench-common";
+import type { InsertableTypedNode, TreeView } from "@fluidframework/tree/internal";
+import { type App, type Bubble, Client } from "./schema.js";
+
+export class AppState implements IAppState {
+	readonly localClient: Client;
+
+	constructor(
+		private readonly tree: TreeView<App>,
+		public width: number,
+		public height: number,
+		numBubbles: number,
+	) {
+		this.tree.root.clients.insertAtEnd(
+			this.createInitialClientNode(numBubbles) as unknown as Client,
+		);
+		// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+		const lastClient = this.tree.root.clients.at(this.tree.root.clients.length - 1)!;
```

Comment:
If you want the last item of an array using at, you can to .at(-1)
If you want the last item of an array without needing a non null assertion, you can do [length -1]

In this case though I think it would make sense to do something like:
this.localClient = new Client(this.createInitialClientNode(numBubbles));
this.tree.root.clients.insertAtEnd(this.localClient);